src/components/TableNew/TablePager.tsx

```diff
@@ -0,0 +1,87 @@
+import { Button } from '@/components/ButtonNew/Button'
+import { Paragraph } from '@/components/TypographyNew'
+import { useEffect, useState } from 'react'
+
+interface TablePagerProps {
+  pageSize: number
+  totalItems: number
+  pagedItemName: string
+  onPageChange: (startIndex: number, endIndex: number) => void
+  mode: 'cyclic' | 'expandable'
+}
+
+export const TablePager: React.FC<TablePagerProps> = ({
+  pageSize,
+  totalItems,
+  pagedItemName,
+  onPageChange,
+  mode,
+}) => {
+  const [currentPage, setCurrentPage] = useState(0)
+  const [expandEnd, setExpandEnd] = useState(pageSize)
```

Comment:
```suggestion
  const [expandedEnd, setExpandedEnd] = useState(pageSize)
```

if you start a var with a verb it makes it look like a function/action